bin/postsToDb.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+let denodeify = require("denodeify");
+let _fs       = require("fs");
+let path      = require("path");
+let pm        = require("promised-mongo");
+
+let db        = pm("mongodb://localhost/bydesign", ["authors", "posts"]);
+
+const fs = {
+	readFile: denodeify(_fs.readFile)
+};
+
+console.log("Starting...");
+
+db.posts.find({})
+	.then((posts) =>
+		posts.forEach((post) => {
+			console.log(`Reading ${post.title.text}...`);
+			return fs.readFile(path.join(__dirname, `../posts/${post.guid}.md`))
+				.then((content) => {
+					console.log(`Updating ${post.title.text}.`);
+					return db.posts.update({ guid: post.guid }, { $set: { content: content.toString() } });
```

Comment:
So. Let's talk about organisation. When I wrote this transfer code the first time, I too stuffed content into the same collection as the post metadata. But that seemed problematic to me. I realised that while posts and post data are inexorably linked, there are many cases in which you would want only the post data and not the post itself, but by linking them this way, you always force a fetch of the data. Now, I know that with a mongo server running on the same backend as the node server, making big queries is less of an issue. At the same time, it adds a lot of overhead to every query which we don't necessarily need

What I ended up doing was creating a separate collection for the posts, and linking them with `ObjectId`s. This means you have negligible lookup times for fetching the post from the metadata, and you also don't have to always grab the whole big content field. This might be an unnecessary optimisation, so I'm open to suggestion/debate.